mlrun/model_monitoring/features_drift_table.py

```diff
@@ -279,6 +278,7 @@ def _plot_feature_row_table(
         input_statistics: dict,
         metrics: dict,
         row_height: int,
+        feature: str,
```

Comment:
add it to the docstring to keep consistency with the other function params

Reply:
actually why do you prefer to pass `feature` here if you already have `feature_name`? perhaps it's better to pass only `feature_name` and change the variable name when parsing it into html format. something like 
`parsed_feature_name = "<br>".join(self._separate_feature_name(feature_name))`

Reply:
Right, great catch! 😅
Done in c6f8e0d.